libmscore/score.cpp

```diff
@@ -2302,113 +2302,124 @@ void Score::splitStaff(int staffIdx, int splitPoint)
       int strack = staffIdx * VOICES;
       int dtrack = (staffIdx + 1) * VOICES;
 
+      // Keep track of ties to be reconnected.
+      struct OldTie {
+            Tie* tie;
+            Note* nnote;
+            };
+      QMap<Note*, OldTie> oldTies;
+
+      // Notes under the split point can be part of a tuplet, so keep track
+      // of the tuplet mapping too!
+      QMap<Tuplet*, Tuplet*> tupletMapping;
+      Tuplet* tupletSrc[VOICES] = { };
+      Tuplet* tupletDst[VOICES] = { };
```

Comment:
Maybe `QList<Tuplet*>` is better?

Reply:
or `QVector`?

Reply:
No, because I need the tuplet for each for voice, or nullptr is there is no current tuplet for a voice at that particular moment.

Reply:
Then initiate it to be `{nullptr, nullptr, nullptr, nullptr}`?